src/main/webapp/WEB-INF/pages/users.jsp

```diff
@@ -11,36 +11,42 @@
+                <th scope="col" class="border-0 text-uppercase font-medium">First Name</th>
+                <th scope="col" class="border-0 text-uppercase font-medium">Last Name</th>
+                <th scope="col" class="border-0 text-uppercase font-medium">Username</th>
+                <th scope="col" class="border-0 text-uppercase font-medium">Email</th>
+                <th scope="col" class="border-0 text-uppercase font-medium">Role</th>
+            </tr>
+        </tbody>
+        <c:set var="count" value="0" scope="page" />
+        <c:forEach var="user" items="${users}" varStatus="status">
+            <tbody>
                 <tr>
-                    <th scope="col" class="border-0 text-uppercase font-medium pl-4">#</th>
-                    <th scope="col" class="border-0 text-uppercase font-medium">First Name</th>
-                    <th scope="col" class="border-0 text-uppercase font-medium">Last Name</th>
-                    <th scope="col" class="border-0 text-uppercase font-medium">Username</th>
-                    <th scope="col" class="border-0 text-uppercase font-medium">Email</th>
-                    <th scope="col" class="border-0 text-uppercase font-medium">Role</th>
+                    <td class="pl-4">
+                        <c:set var="count" value="${count + 1}" scope="page"/>
+                        <c:out value="${count}" />
+                        <%--${user.id}--%>
+                    </td>
+                    <td>
+                        ${user.firstName}
+                    </td>
+                    <td>
+                        ${user.lastName}
+                    </td>
+                    <td>
+                        ${user.username}
+                    </td>
+                    <td>
+                        ${user.email}
+                    </td>
+                    <td>
+                        ${user.position}
+                    </td>
                 </tr>
-    <c:set var="count" value="0" scope="page" />
-    <c:forEach var="user" items="${users}" varStatus="status">
-    <tr>
-                            <td class="pl-4">
-                                <c:set var="count" value="${count + 1}" scope="page"/>
-                                <c:out value="${count}" />
-                                <%--${user.id}--%>
-                            </td>
-                            <td>
-                                ${user.firstName}
-                            </td>
-                            <td>
-                                ${user.lastName}
-                            </td>
-                            <td>
-                                ${user.username}
-                            </td>
-                            <td>
-                                ${user.email}
-                            </td>
-                            <td>
-                                ${user.position}
-                            </td>
-    </c:forEach>
+            </c:forEach>
```

Comment:
wrong order of closing tags, it should be like this

`    </tbody>
</c:forEach>`